controllers/metallb_controller.go

```diff
@@ -156,6 +156,12 @@ func (r *MetalLBReconciler) syncMetalLBResources(config *metallbv1beta1.MetalLB)
 	data.Data["NameSpace"] = r.Namespace
 	data.Data["KubeRbacProxy"] = os.Getenv("KUBE_RBAC_PROXY_IMAGE")
 	data.Data["DeployKubeRbacProxies"] = os.Getenv("DEPLOY_KUBE_RBAC_PROXIES") == "true"
+
+	data.Data["LogLevel"] = "info"
```

Comment:
Can we have a validation of the value of the loglevel (maybe as a webhook)?
If not, it will make metallb crash

Reply:
Sure, going to use an enum to get it more robust